Pear/Profile/EditProfileSectionsViewController.swift

```diff
@@ -7,6 +7,22 @@
 
 import UIKit
 
+protocol didUpdateInterestsDelegate: AnyObject {
+    func updateInterests(updatedUser: UserV2, newInterests: [Interest])
+}
+
+protocol didUpdateGroupsDelegate: AnyObject {
+    func updateGroups(updatedUser: UserV2, newGroups: [Group])
+}
+
+protocol didUpdatePromptsDelegate: AnyObject {
+    func updatePrompts(updatedUser: UserV2, newPrompt: Prompt)
+}
+
+protocol didCreateNewPrompt: AnyObject {
```

Comment:
You could probably just have 1 protocol, `EditProfileDelegate` here with four different methods, one for updating interests, groups, prompts, and adding a new prompt. Also protocols should also start with a capitalized letter

Reply:
Did you address this?

Reply:
Hey Reade - for this one, I consolidated updating interests, groups, and prompts but I kept a separate protocol for creating prompts since it's a bit of a different process from updating prompts.